src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -62,8 +61,12 @@ public BulkInsertCommand(long id, BulkInsertStreamExposerContent stream, string
                 Timeout = TimeSpan.FromHours(12); // global max timeout
             }
 
+            internal string RequestNodeTag { get; private set; }
+
             public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
             {
+                RequestNodeTag = node.ClusterTag;
```

Comment:
why not to use `SelectedNodeTag` ?

Reply:
`SelectedNodeTag` is set only if you do it explicitly in the command. We don't define it explicitly so I'm getting the actual node where the request was sent so next I can use it to kill the operation (if any error is thrown).

https://github.com/ravendb/ravendb/pull/13962/files#diff-b33c28b61bd3a1c5bb3a521dc70f29b005d147768ba081b1be538006b0fae5adR60